jdaviz/configs/specviz/helper.py

```diff
@@ -128,18 +128,23 @@ def get_spectra(self, data_label=None, subset_to_apply=None, apply_slider_redshi
 
             return output_spectra
 
-    def get_spectral_regions(self):
+    def get_spectral_regions(self, use_display_units=False):
```

Comment:
we might want to discuss the default option here and in `get_subsets` (but can change that anytime before merging the dev unit conversion branch into main)